xds/internal/balancer/wrrlocality/balancer_test.go

```diff
@@ -119,3 +130,135 @@ func (s) TestParseConfig(t *testing.T) {
 		})
 	}
 }
+
+func setup(t *testing.T) (*wrrLocalityBalancer, func()) {
```

Comment:
Could you use a more descriptive name please.

Reply:
Also, do you expect more tests that would be using this helper? If not, you might as well get rid of it and inline it in the test.